### ISSUE TITLE
Drop table isn't catched

### ISSUE THREAD

**Issue by POD666** [3YOURMIND/django-migration-linter#70]
Let's say we have the following migration:
```
operations = [
        migrations.DeleteModel(
            name='A',
        ),
    ]
```
It's not a backward compatible change so should be catched by linter but it's not.
```
(app_drop_table, 0001_initial)... OK
(app_drop_table, 0002_delete_a)... OK
```
